mathics/builtin/box/graphics.py

```diff
@@ -363,8 +363,8 @@ class CircleBox(_ArcBox):
     </dl>
     """
 
-    summary_text = "internal box representation for 'Circle' elements"
     face_element = False
+    summary_text = "internal box representation for 'Circle' elements"
```

Comment:
Right now,  in Django _none_ of the entries under "Boxing Routines for 2D Graphics" appears in Django. 

And I am seeing summaries such as under "Drawing Graphics" I am seeing these "box representation" summaries that are truncated. 

Something weird is going on here. 

However in answer to your question, It looks like in summaries we do not do any sort of format tagging like we do in the docstring of the class.

Reply:
@rocky, I wanted to have these summaries for the sake of completeness. Also, you could, inside a Mathics-django session, to define a MakeBoxes rule that uses these symbols, and then it could appear in a tool.
Since all of these summaries are almost always of the same form, I would like to automatize them, but that is still a work in progress...

Reply:
> Not sure if quotes is necessary here. @rocky ?

@TiagoCavalcante, here I followed the convention in the documentation that symbol names are between single quotes.